examples/offline_inference/eagle.py

```diff
@@ -92,30 +93,37 @@ def main():
 
     sampling_params = SamplingParams(temperature=args.temp, max_tokens=256)
 
-    outputs = llm.generate(prompt_token_ids=prompt_ids,
-                           sampling_params=sampling_params)
+    _ = llm.generate(prompt_token_ids=prompt_ids,
+                     sampling_params=sampling_params)
 
-    if not hasattr(outputs, "metrics") or outputs.metrics is None:
+    try:
+        metrics = llm.get_metrics()
+    except AssertionError:
+        # Presumably V0 engine, metrics not supported
```

Comment:
Can we just add a warning message here:
```suggestion
        print("Metrics are not supported in the V0 engine.")
```